tzrec/models/rank_model.py

```diff
@@ -42,15 +42,17 @@ class RankModel(BaseModel):
         model_config (ModelConfig): an instance of ModelConfig.
         features (list): list of features.
         labels (list): list of label names.
+        sample_weights (list): sample weight names.
     """
 
     def __init__(
         self,
         model_config: model_pb2.ModelConfig,
         features: List[BaseFeature],
         labels: List[str],
+        sample_weights: List[str] = None
     ) -> None:
-        super().__init__(model_config, features, labels)
+        super().__init__(model_config, features, labels, sample_weights)
         self._num_class = model_config.num_class
```

Comment:
add sample weight for loss of rank and multi_task_rank models later?

Reply:
rank models will be fixed in the next PR.